test/scripts/start-services.sh

```diff
@@ -35,10 +35,15 @@ start_geth() {
 
     local data_dir="$output_dir/geth"
 
+    # removes TTD bomb for when beacon chain is not used
+    if [ "$start_beacon_sync" == "false" ]; then
+        jq 'del(.config.terminalTotalDifficulty)' "$output_dir/genesis.json" | sponge "$output_dir/genesis.json"
+    fi
+    
     if [ "$eth_network" == "localhost" ]; then
         echo "Starting geth local net"
 
-        geth init --datadir "$data_dir" config/genesis.json
+        geth init --datadir "$data_dir" "$output_dir/genesis.json"
```

Comment:
I changed it to use the modified file.

Reply:
Ah nice find! This is why removing TTD didnt work for me!